resources/views/auth/passwords/reset.blade.php

```diff
@@ -1,75 +1,69 @@
 @extends('layouts.main_layout')
 
 @section('head')
-    <!-- Redirect to main page if user is already logged in -->
-    @unless(Auth::guest())
-        <script>window.location.href = "/projects";</script>
-    @endunless
-
     <title>Password Reset</title>
 @endsection
 
 @section('mainBody')
 
-    <!-- Main -->
-
-    <main class="primary-main row">
-        <section class="grid login-main">
-
-            <!-- Login -->
-
-            <div class="col-1-3">
-
-                <form class="login-form" action="{{ url('/password/reset') }}" method="POST">
-                    {!! csrf_field() !!}
-
-                    <input type="hidden" name="token" value="{{ $token }}">
-
-                    <h5>Reset Password</h5>
-
-                    @if (session('status'))
-                        <div class="alert alert-success">
-                            {{ session('status') }}
+<main class="primary-main row center-block">
+    <div class="jumbotron col-sm-6 col-sm-offset-3">
+        <form class="form-horizontal" action="{{ url('/password/reset') }}" method="POST">
+            {!! csrf_field() !!}
+
+            <input type="hidden" name="token" value="{{ $token }}">
+            <fieldset>
+                <legend>Reset Password</legend>
+
+                <div class="form-group">
+                    <!--
+                    @if($errors->has('email'))
+                        <div class="has-error">
+                    @endif -->
+                    <label for="email" class="col-lg-2 control-label">Email</label>
+                    <div class="col-lg-10">
+                        <input class="form-control" type="email" name="email" id="email" value="{{ $email or old('email') }}">
+                        <!--@if ($errors->has('email'))
+                            <label class="control-label">{{ $errors->first('email') }}</label>
+                            </div>
+                        @endif -->
+                    </div>
+                </div>
+                @if($errors->has('password') or $errors->has('password_confirmation'))
+                    <div class="has-error">
```

Comment:
Can you line this error message up on the left like the other error messages.
![error1](https://cloud.githubusercontent.com/assets/16866862/14696762/a474b66e-0732-11e6-964e-50432073eca0.PNG)